textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/FuzzyTokenizer.java

```diff
@@ -0,0 +1,38 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ * Class to take in a string and uses a dictionary of words, to split it into possible
+ * set of strings, with appropriate space delimitions. his is a resource used by the QueryRewriter
+ * operator
+ *
+ *
+ * For example - "newyorkcity" - > ["new york city", "newyorkcity"]
+ * For example - "horseshoe" -> ["hor se shoe", "hors es hoe", "horse shoe", "horses hoe", "horseshoe"]
+ */
+public class FuzzyTokenizer {
```

Comment:
Understood.  Again, please remove the implementation of the operator from this PR.  You can include it in the next PR.